packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
 ---
-title: Internal Tools
+title: Access internal tools
 ---
 
 # Internal Tools
 
-This document describes tools available internally to the vets.gov team and configuration required to access them. You'll need access to these tools if you need:
+These instructions cover the configuration of a SOCKS proxy necessary for access to:
 
-* Build logs and details from Jenkins (linked to from GitHub PRs for each project)
+* Static `vets-website` content
+* Remote build logs for pull requests and deployments (Jenkins)
```

Comment:
Made this more specific since it doesn't include local build logs and didn't assume that front ends know what Jenkins is for.